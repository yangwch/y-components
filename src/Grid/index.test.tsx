import React from 'react';
import { render } from '@testing-library/react';
import { mountTest } from '../../test/shared/mountTest';

import { Row, Col } from './index';

describe('Button', () => {
  mountTest(() => <Row></Row>);
  mountTest(() => (
    <Row wrap={false}>
      <Col></Col>
    </Row>
  ));
  mountTest(() => (
    <Row>
      <Col span={1}></Col>
      <Col span={2}></Col>
    </Row>
  ));

  it('render correctly', () => {
    const renderFn = () =>
      render(
        <Row>
          <Col>test</Col>
        </Row>,
      );
    expect(renderFn).not.toThrowError();

    const { container } = renderFn();
    expect(container.firstChild).toMatchSnapshot();
  });
});
