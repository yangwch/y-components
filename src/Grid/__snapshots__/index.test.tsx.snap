// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button render correctly 1`] = `
<div
  class="yy-row"
>
  <div
    class="yy-col"
  >
    test
  </div>
</div>
`;
